### PR TITLE
chore: Add gallery projects under examples

### PR DESCRIPTION
**Pull request by ssheng**
## Before submitting:

- [ ] Does the Pull Request follow [Conventional Commits specification](https://www.conventionalcommits.org/en/v1.0.0/#summary) naming? Here are [GitHub's
guide](https://docs.github.com/en/pull-requests/collaborating-with-pull-requests/proposing-changes-to-your-work-with-pull-requests/creating-a-pull-request) on how to create a pull request.
- [ ] Does the code follow BentoML's code style, both `make format` and `make lint` script have passed ([instructions](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#style-check-auto-formatting-type-checking))?
- [ ] Did you read through [contribution guidelines](https://github.com/bentoml/BentoML/blob/main/CONTRIBUTING.md#ways-to-contribute) and follow [development guidelines](https://github.com/bentoml/BentoML/blob/main/DEVELOPMENT.md#start-developing)?
- [ ] Did your changes require updates to the documentation? Have you updated
  those accordingly? Here are [documentation guidelines](https://github.com/bentoml/BentoML/tree/main/docs) and [tips on writting docs](https://github.com/bentoml/BentoML/tree/main/docs#writing-documentation).
- [ ] Did you write tests to cover your changes?

## Who can help review?

Feel free to tag members/contributors who can help review your PR.

### CODE REVIEW

File: examples/README.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+[![BentoML](https://user-images.githubusercontent.com/489344/178160978-6e13a849-d16a-43b4-9ada-ef30f35922fa.png)](http://bentoml.com)
+
+# BentoML Gallery 🎨 [![Twitter Follow](https://img.shields.io/twitter/follow/bentomlai?style=social)](https://twitter.com/bentomlai) [![Slack](https://img.shields.io/badge/Slack-Join-4A154B?style=social)](https://l.linklyhq.com/l/ktO8)
```
Comment (aarnphm) [inline examples/README.md:3]:
```suggestion
# BentoML Examples 🎨 [![Twitter Follow](https://img.shields.io/twitter/follow/bentomlai?style=social)](https://twitter.com/bentomlai) [![Slack](https://img.shields.io/badge/Slack-Join-4A154B?style=social)](https://l.linklyhq.com/l/ktO8)
```

File: examples/README.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+
+BentoML is an open platform for machine learning in production. It simplifies model packaging and model management, optimizes model serving workloads to run at production scale, and accelerates the creation, deployment, and monitoring of prediction services.
+
+The gallery repository contains a collection of example projects demonstrating [BentoML](https://github.com/bentoml/BentoML)
+usage and best practices.
+
+👉 [Pop into our Slack community!](https://join.slack.bentoml.org) We're happy to help with any issue you face or even just to meet you and hear what you're working on :)
```
Comment (aarnphm) [inline examples/README.md:10]:
```suggestion
```

File: examples/release_quickstart_image.sh [hunk 3]
```diff
@@ -0,0 +1,40 @@
+#!/bin/bash
```
Comment (aarnphm) [inline examples/release_quickstart_image.sh:1]: we can move this to ./scripts

**Review by aarnphm — APPROVED** [review]
minor tings.

File: examples/README.md (hunk 1 above)
Comment (ssheng) [reply examples/README.md]: Updated in the latest iteration.

File: examples/README.md (hunk 2 above)
Comment (ssheng) [reply examples/README.md]: Updated in the latest iteration.

File: examples/release_quickstart_image.sh (hunk 3 above)
Comment (ssheng) [reply examples/release_quickstart_image.sh]: Good idea. Moved and updated the docker script.

**Pull request merged**